packages/gas-fee-controller/src/GasFeeController.test.ts

```diff
@@ -48,39 +50,46 @@ const mockedDetermineGasFeeCalculations =
 
 const name = 'GasFeeController';
 
-type MainMessenger = Messenger<
-  | GetGasFeeState
-  | NetworkControllerGetStateAction
-  | NetworkControllerGetNetworkClientByIdAction
-  | NetworkControllerGetEIP1559CompatibilityAction,
-  GasFeeStateChange | NetworkControllerNetworkDidChangeEvent
->;
+type AllGasFeeControllerActions = MessengerActions<GasFeeMessenger>;
+type AllGasFeeControllerEvents = MessengerEvents<GasFeeMessenger>;
+
+type AllNetworkControllerActions = MessengerActions<NetworkControllerMessenger>;
+type AllNetworkControllerEvents = MessengerEvents<NetworkControllerMessenger>;
+
+type AllActions = AllGasFeeControllerActions | AllNetworkControllerActions;
+type AllEvents = AllGasFeeControllerEvents | AllNetworkControllerEvents;
 
-const getMessenger = (): MainMessenger => {
-  return new Messenger();
+type RootMessenger = Messenger<MockAnyNamespace, AllActions, AllEvents>;
```

Comment:
Nit: Do we need to assign the actions and events for messengers to type aliases?  We don't seem to do this in other places.

``` suggestion
type AllActions =
  | MessengerActions<GasFeeMessenger>
  | MessengerActions<NetworkControllerMessenger>;

type AllEvents =
  | MessengerEvents<GasFeeMessenger>
  | MessengerEvents<NetworkControllerMessenger>;

type RootMessenger = Messenger<MockAnyNamespace, AllActions, AllEvents>;